Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM registry.redhat.io/ubi8/python-39
```

Comment:
`registry.access.redhat.com/ubi8/python-39:latest` Shouldn't be used this for unauthorized access? I cannot build image without auth.

Reply:
Also to make Handolint happy and prevent breaking changes we should use more specific floating tag `python-39:1`

Reply:
Done!

(Oddly enough, it wouldn't let me get this image from this registry before. Or maybe that was a slightly different image reference)